menu/cbs/menu_cbs_title.c

```diff
@@ -24,7 +24,7 @@
 
 #include "../../retroarch.h"
 #include "../../configuration.h"
-#include "../managers/core_option_manager.h"
+#include "../../managers/core_option_manager.h"
```

Comment:
This is bonus, the path is wrong and just happens to work because of `-I./deps/`. Its better to not depend on this and just use the correct path.